ideascube/search/migrations/0002_reindex.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, connections
-from ideascube.search.utils import reindex_content
-
-
-def update_reindex(*args):
-    cursor_default = connections['default'].cursor()
-    cursor_default.execute("DROP TABLE IF EXISTS idx")
```

Comment:
This is not done anymore.
Maybe move it in utils.create_index_table ? Even if an **create**_index_table should not make migration or delete table.